packages/opentelemetry-plugin-document-load/src/documentLoad.ts

```diff
@@ -178,6 +205,25 @@ export class DocumentLoad extends BasePlugin<unknown> {
     return entries;
   }
 
+  /**
+   * Creates and ends a span with network information about resource added as timed events
+   * @param rootSpan
+   * @param resource
+   * @private
+   */
+  private _initResourceSpan(
+    rootSpan: Span,
+    resource: PerformanceResourceTiming
+  ) {
+    const span = this._startSpan(resource.name, PTN.FETCH_START, resource, {
+      parent: rootSpan,
+    });
+    if (span) {
```

Comment:
Could this ever be not defined?

Reply:
yes if the certain metric doesn't exist (for any reason), then the startSpan will never create a new span